cmd/openshift-install/create.go

```diff
@@ -186,6 +193,49 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 	}
 }
 
+// appendRouterCAToClusterCA appends router CA to cluster CA in kubeconfig
+func appendRouterCAToClusterCA(config *rest.Config, directory string) (err error) {
+	client, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return errors.Wrap(err, "creating a Kubernetes client")
+	}
+
+	routerSecret, err := client.CoreV1().Secrets("openshift-ingress-operator").Get("router-ca", metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		return err
+	}
+	routerCrtBytes := routerSecret.Data["tls.crt"]
+	absDir, err := filepath.Abs(directory)
+	if err != nil {
+		return err
+	}
+	kubeconfig := filepath.Join(absDir, "auth", "kubeconfig")
+	yml, err := ioutil.ReadFile(kubeconfig)
+	if err != nil {
+		return err
+	}
+	var kconfig *clientcmdv1.Config
+	err = yaml.Unmarshal(yml, &kconfig)
```

Comment:
It would be nice to grab the `kubeconfig.Admin` asset from the store, which would already have the unmarshaled config. Unfortunately, (1) the asset store used has already gone out of scope, and (2) there is currently no way of fetching from the store without purging the other assets from disk.

Reply:
right, I pondered that